index.js

```diff
@@ -51,9 +55,21 @@ function wrap(opts) {
 			return transform(input, metadata);
 		}
 
-		var hash = hashFn(input, metadata, salt);
+		var data = [ownHash || getOwnHash(), input];
+		if (salt) {
+			data.push(salt);
+		}
```

Comment:
newlines before and after if blocks.

I forget to follow this myself half the time. But @sindresorhus is going to introduce it into XO as soon as the rule exists, so might as well start following the rule.